src/App/Formatter/Post/CSV.php

```diff
@@ -487,6 +495,27 @@ protected function isLocation($value)
             array_key_exists('lat', $value);
     }
 
+    /**
+     * Decode checkbox value
+     *
+     * @param mixed $value
+     *
+     * @return array
+     */
+    protected function decodeCheckboxValue($value)
```

Comment:
@webong  Would the result of this be that checkboxes are now exported into CSV encoded to JSON? In that case, this could get confusing to folks using the CSV

Reply:
Not at all, after it's converted to json there's an extra formatter that uses the explode function in php to convert the object to a series of string.